enterprise/views.py

```diff
@@ -590,9 +591,16 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
             )
         except (AttributeError, ValueError):
             course_effort = ''
-        organization = organizations_helpers.get_organization(course_details['org'])
         course_run = CourseCatalogApiClient(request.user).get_course_run(course_details['course_id'])
 
+        try:
+            organization = organizations_helpers.get_organization(course_details['org'])
+            organization_logo = organization['logo'].url
```

Comment:
@replaceafill There's one more edge case here that we should probably handle: To test this change I deleted the logo from the organization I had created in the Django admin (by checking "clear" next to the current logo and clicking "Save"). When I tried to access the landing page after that this line errored out with:

> `ValueError`: The 'logo' attribute has no file associated with it.